packages/office-ui-fabric-react/src/components/ComboBox/__snapshots__/ComboBox.test.tsx.snap

```diff
@@ -45,7 +45,7 @@ exports[`ComboBox Renders ComboBox correctly 1`] = `
         &:hover {
           border-color: #0078d7;
         }
-        @media screen and (-ms-high-contrast: active) & {
+        @media screen and (-ms-high-contrast: active){&:hover {
```

Comment:
This is pretty awesome! Changes in merge-styles now reflect in the snapshot tests!